components/bl-collapsiblePanel-component/README.md

```diff
@@ -0,0 +1,76 @@
+# CollapsiblePanel
+
+CollapsiblePanel is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). This component displays summary information and upon selecting the panel, it expands to display the full expansion panel.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property                        | Type   | Default value | Logic              | Data Binding | UI Setting | Description                                           |
+|---------------------------------|--------|---------------|--------------------|--------------|------------|-------------------------------------------------------|
+| Title <br> `title`              | *Text* | `Panel Title` | Title Logic        | YES          | YES        | The logic to determine the title of the panel.        |
+
+## Events
+
+| Name                 | Triggers                                | Context Blocks |
+|----------------------|-----------------------------------------|----------------|
+| On Open Panel Event  | when the user opens the panel section.  |                |
+| On Close Panel Event | when the user closes the panel section. |                |
+
+## Actions
+
+| Action                           | Inputs | Returns |
+|----------------------------------|--------|---------|
+| Show Content Of CollapsiblePanel |        |         |
```

Comment:
` On Open Panel Event` vs `Show Content Of CollapsiblePanel` sound like something for different components and

what about to have:

```
On Expand/Collapse Event - events

Expand/Collapse Content - actions
```

what do you think?

Reply:
Good idea, agree. I'll change it.